src/aabb.rs

```diff
@@ -155,6 +155,52 @@ impl AABB {
         (p.z - self.min.z) > -epsilon && (p.z - self.max.z) < epsilon
     }
 
+    /// Returns true if the `other` [`AABB`] is approximately inside this [`AABB`]
+    /// with respect to some `epsilon`.
+    ///
+    /// # Examples
+    /// ```
+    /// use bvh::EPSILON;
+    /// use bvh::aabb::AABB;
+    /// use bvh::nalgebra::Point3;
+    ///
+    /// let aabb = AABB::with_bounds(Point3::new(-1.0, -1.0, -1.0), Point3::new(1.0, 1.0, 1.0));
+    /// let point_barely_inside = Point3::new(1.00_000_01, 1.00_000_01, 1.00_000_01);
+    /// let center = aabb.center();
+    /// let inner_aabb = AABB::with_bounds(center, point_barely_inside);
+    ///
+    /// assert!(aabb.approx_contains_aabb_eps(&inner_aabb, EPSILON));
+    /// ```
+    ///
+    /// [`AABB`]: struct.AABB.html
+    pub fn approx_contains_aabb_eps(&self, other: &AABB, epsilon: f32) -> bool {
+        self.approx_contains_eps(&other.min, epsilon) &&
+        self.approx_contains_eps(&other.max, epsilon)
+    }
+
+    /// Returns true if the `other` [`AABB`] is approximately equal to this [`AABB`]
+    /// with respect to some `epsilon`.
+    ///
+    /// # Examples
+    /// ```
+    /// use bvh::EPSILON;
+    /// use bvh::aabb::AABB;
+    /// use bvh::nalgebra::Point3;
+    ///
+    /// let aabb = AABB::with_bounds(Point3::new(-1.0, -1.0, -1.0), Point3::new(1.0, 1.0, 1.0));
+    /// let point_barely_inside_min = Point3::new(-1.00_000_01, -1.00_000_01, -1.00_000_01);
+    /// let point_barely_inside_max = Point3::new(1.00_000_01, 1.00_000_01, 1.00_000_01);
+    /// let other = AABB::with_bounds(point_barely_inside_min, point_barely_inside_max);
+    ///
+    /// assert!(aabb.relative_eq(&other, EPSILON));
+    /// ```
+    ///
+    /// [`AABB`]: struct.AABB.html
+    pub fn relative_eq(&self, other: &AABB, epsilon: f32) -> bool {
```

Comment:
This will stay this way, because the approx crate has the same naming scheme.
in approx, relative_eq is an imprecise floating point comparison.
Then we traverse nalgebra, which uses approx to define imprecise fpoint comparison on vectors with `relative_eq`.
Our AABBs adhere to the same comparison rules.